pcsx2/gui/Panels/GameDatabasePanel.cpp

```diff
@@ -430,15 +434,21 @@ bool Panels::GameDatabasePanel::WriteFieldsToDB() {
 	if (serialBox->GetValue().IsEmpty()) return false;
 
 	Game_Data game;
+	wxString com = compatBox->GetValue();
 	GameDB->findGame(game, serialBox->GetValue());
 
 	game.id = serialBox->GetValue();
 
-	game.writeString(L"Serial",		serialBox->GetValue());
-	game.writeString(L"Name",		nameBox->GetValue());
-	game.writeString(L"Region",		regionBox->GetValue());
-	game.writeString(L"Compat",		compatBox->GetValue());
-	game.writeString(L"[comments]",	commentBox->GetValue());
+	game.writeString(L"Serial", serialBox->GetValue());
+	game.writeString(L"Name", nameBox->GetValue());
+	game.writeString(L"Region", regionBox->GetValue());
+	if (com == "Unknown")          game.writeString(L"Compat", "0");
```

Comment:
I think we're getting a bit sidetracked on the switch and string stuff.  In this case doing a string comparison whatever way is wrong.

Using a combobox (as I suggested before) is more appropriate. The user should just be able to select whether it's payable/menyu/in-ghame/whatever instead of having to actually type it out - it's much less error prone.